scripts/system/create/entitySelectionTool/entitySelectionTool.js

```diff
@@ -749,216 +749,463 @@ SelectionDisplay = (function() {
 
     that.replaceCollisionsAfterStretch = false;
 
+    var toolEntitiesMaterialData = "{\n  \"materialVersion\": 1,\n  \"materials\": [\n    {\n      \"name\": \"0\",\n      \"albedo\": \"fallthrough\",\n      \"unlit\": true,\n      \"model\": \"hifi_pbr\"\n    }\n  ]\n}";
```

Comment:
instead of setting albedo to fallthrough, this should set defaultFallthrough to true